sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/HiveSessionImpl.java

```diff
@@ -171,7 +172,7 @@ protected BufferedReader loadFile(String fileName) throws IOException {
       FileInputStream initStream = null;
       BufferedReader bufferedReader = null;
       initStream = new FileInputStream(fileName);
-      bufferedReader = new BufferedReader(new InputStreamReader(initStream));
+      bufferedReader = new BufferedReader(new InputStreamReader(initStream, StandardCharsets.UTF_8));
```

Comment:
the code here is copied from Hive, do we have the same issue in the upstream repo? Better to cite here

Reply:
OK，I would address it @yaooqinn

Reply:
Yeah, I wouldn't touch here.